xml_inscriptions/transcribed/MI.AA.UM.KM.L.1008.xml

```diff
@@ -108,31 +108,27 @@
         <div type="edition">
             <ab>
                 <lb n="1"/>
-               <supplied reason="lost">
-                  <expan>
-                     <abbr>D</abbr><ex>is</ex>
-                  </expan>
-               </supplied> 
+               <supplied reason="lost"><expan><abbr>D</abbr><ex>is</ex></expan></supplied> 
                <expan>
                   <abbr>M</abbr><ex>anibus</ex></expan>.
                 <lb n="2"/> 
-               <gap reason="lost" extent="unknown" unit="character"/><unclear>L</unclear>O <persName>
+               <gap reason="lost" extent="unknown" unit="character"/><unclear>l</unclear>o <persName>
```

Comment:
My understanding is that convention is to leave these letters capitalized because they are uninterpretable in their contexts?